src/modules/NtpClient.rb

```diff
@@ -839,19 +853,35 @@ def check_service
       end
     end
 
-    # If synchronize time has been enable it writes ntp cron entry for manual
-    # sync. If not it removes current cron entry if exists.
-    def update_cron_settings
+    # If synchronize time has been enable it writes systemd timer entry for manual
+    # sync. If not it removes current systemd timer entry if exists.
+    def update_timer_settings
       if @synchronize_time
         SCR.Write(
           path(".target.string"),
-          CRON_FILE,
-          "-*/#{@sync_interval} * * * * root /usr/sbin/chronyd -q &>/dev/null\n"
+          TIMER_PATH,
+          format(TIMER_CONTENT, timeout: @sync_interval)
+        )
+        SCR.Execute(
```

Comment:
Again, no support in YaST for this? I'd avoid directly calling `systemctl` if possible.

Reply:
well, we have Systemctl class, but in general now we only support service, sockets and targets, not timers.

Reply:
Ah, I see, OK.